docs/standards/smirnoff.md

```diff
@@ -566,15 +566,15 @@ Improper torsions are specified via an `<ImproperTorsions>...</ImproperTorsions>
     ...
 </ImproperTorsions>
 ```
-Currently, only `potential="charmm"` is supported, where we utilize the functional form:
+Currently, only `potential="k*(1+cos(periodicity*theta-phase))"` is supported, where we utilize the functional form:
```

Comment:
IMO it would be nice to still communicate that this is "CHARMM-style", since I think CHARMM vs. Amber improper is a common point of confusion. I don't care about the phrasing/location as long as that point comes through
```suggestion
Currently, only `potential="k*(1+cos(periodicity*theta-phase))"` is supported, where we utilize the functional form of CHARMM:
```